brigade-worker/src/index.ts

```diff
@@ -84,7 +84,8 @@ if (script) {
   require(script);
 }
 
-const logLevel = LogLevel[process.env.BRIGADE_LOG_LEVEL || "LOG"];
+// Log level may come in as lowercased 'log', 'info', etc., if run by the brig cli
+const logLevel = LogLevel[process.env.BRIGADE_LOG_LEVEL.toUpperCase() || "LOG"];
```

Comment:
When the environment variable is not set, the attempt to execute `toUpperCase()` results in throwing an undefined error:

```
const logLevel = logger_1.LogLevel[process.env.BRIGADE_LOG_LEVEL.toUpperCase() || "LOG"];
                                                                 ^

[  TypeError: Cannot read property 'toUpperCase' of undefined                                                                                                                    
  - index.js:76 Object.<anonymous>                                                                                                                                                   /home/radu/projects/src/github.com/brigadecore/brigade/brigade-worker/dist/index.js:76:66                                                                                    
  - loader.js:774 Module._compile                                                                                                                                                    internal/modules/cjs/loader.js:774:30                                                                                                                                        
  - loader.js:785 Object.Module._extensions..js                                                                                                                                      internal/modules/cjs/loader.js:785:10                                                                                                                                        
  - loader.js:641 Module.load                                                                                                                                                        internal/modules/cjs/loader.js:641:32                                                                                                                                        
  - loader.js:556 Function.Module._load                                                                                                                                              internal/modules/cjs/loader.js:556:12                                                                                                                                        
  - loader.js:837 Function.Module.runMain                                                                                                                                            internal/modules/cjs/loader.js:837:10                                                                                                                                        
  - run_main_module.js:17                                                                                                                                                            internal/main/run_main_module.js:17:11                                                                                                                                       
]
```